### PR TITLE
[9.0][ADD] purchase request to requisition

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
# Purchase Request to Call for Bids

This module introduces the possibility to create new Call for Bids or update existing Bids from Purchase Request Lines. 
This module depends on purchase request, which is used to give notification of requirements of materials. The PR for this one is here: https://github.com/OCA/purchase-workflow/pull/259
### Usage
- Go to Purchase Request Lines and Press 'Create Purchase Bid'. You can choose to select an existing
  Draft Bid or create a new one.
- In case that you chose to select an existing Bid, the application will search for existing lines matching the request line, and will add the extra quantity to them.
- In case that you create a new RFQ, the request lines will also be consolidated into as few as possible lines in the RFQ.

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
cc @jbeficent

**Comment by adrienpeiffer** [comment]
:+1: (Functional tests & code review)

File: purchase_request_to_requisition/models/product.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    @api.multi
+    @api.constrains('purchase_request', 'purchase_requisition')
+    def _check_request_requisition(self):
+        for product in self:
+            if product.purchase_request and product.purchase_requisition:
```
Comment (adrienpeiffer) [inline purchase_request_to_requisition/models/product.py]: After reflexion, `product.purchase_requisition` is a radio button selection in version 9.0. Then, the second part of your condition will always be true. So, it's not possible to select `purchase_request`

Comment (AaronHForgeFlow) [reply purchase_request_to_requisition/models/product.py]: You're right. Thanks!

**Comment by adrienpeiffer** [comment]
:+1: (Code review and functional tests)
Thanks !

**Comment by pedrobaeza** [comment]
Please check Travis status.

**Review by sbidoul — APPROVED** [review]
(code review only)

**Review by sbidoul — APPROVED** [review]
(code review only)

**Review by JordiBForgeFlow — APPROVED** [review]
:+1:

**Pull request merged**